pandas/core/panel.py

```diff
@@ -944,27 +944,28 @@ def construct_index_parts(idx, major=True):
 
     def apply(self, func, axis='major', **kwargs):
         """
-        Applies function along input axis of the Panel
+        Applies function along axis (or axes) of the Panel
 
         Parameters
         ----------
         func : function
             Function to apply to each combination of 'other' axes
-            e.g. if axis = 'items', then the combination of major_axis/minor_axis
-            will be passed a Series
-        axis : {'major', 'minor', 'items'}
+            e.g. if axis = 'items', the combination of major_axis/minor_axis
+            will each be passed as a Series; if axis = ('items', 'major'), DataFrames
+            of items & major axis will be passed
+        axis : {'items', 'minor', 'major'}, or {0, 1, 2}, or a tuple with two axes
         Additional keyword arguments will be passed as keywords to the function
 
         Examples
         --------
-        >>> p.apply(numpy.sqrt) # returns a Panel
-        >>> p.apply(lambda x: x.sum(), axis=0) # equiv to p.sum(0)
-        >>> p.apply(lambda x: x.sum(), axis=1) # equiv to p.sum(1)
-        >>> p.apply(lambda x: x.sum(), axis=2) # equiv to p.sum(2)
+        >>> panel.apply(np.sqrt) # returns a Panel
+        >>> panel.apply(lambda x: x.sum(), axis=1) # equivalent to p.sum(1), returning DataFrame
+        >>> panel.apply(lambda x: x.sum(), axis='minor') # equivalent to previous
+        >>> panel.apply(lambda x: x.shape, axis=(0,1)) # return the shapes of each DataFrame over axis 2, in Series
```

Comment:
to avoid going to long, add on a separate line (e.g. comment before), skipping a line between examples